resources/[freeroam]/OGPcheats/Code/backend.lua

```diff
@@ -0,0 +1,204 @@
+hold =[[
+------------------------------------------------------------------------------------------------
+--Name: aCore ----------------------------------------------------------------------------------
+--Dev: Atton Risk   ----------------------------------------------------------------------------
+--Status: Development Finished and Public ------------------------------------------------------
+--Year: 2013/2014/2019 -------------------------------------------------------------------------
+--Usage: aCore for OGP only  -------------------------------------------------------------------
+--Version: v1.4 --------------------------------------------------------------------------------
+------------------------------------------------------------------------------------------------
+]]
+
+f,d,a = 112,219,244 
```

Comment:
Was replaced by the following since there was no reason to have those values outside of the function itself. 
`function getClientInfo ()
	local GIRED,GIGREEN,GIBLUE = 112,219,244 
	local clientIp, clientName, clientSerial = getPlayerIP(client), getPlayerName(client), getPlayerSerial(client)
	outputChatBox(tostring(na),client,GIRED,GIGREEN,GIBLUE)
	outputChatBox(tostring(se),client,GIRED,GIGREEN,GIBLUE) 
	outputChatBox(tostring(ip),client,GIRED,GIGREEN,GIBLUE) 
end`